packages/tab-bar/index.js

```diff
@@ -63,11 +63,12 @@ export default class TabBar extends Component {
       getPreviousActiveTabIndex: () => this.state.previousActiveIndex,
       getFocusedTabIndex: () => {
         const activeElement = document.activeElement;
-        this.tabList_.forEach((tabList, index) => {
-          if (tabList.tabElement_.current === activeElement) {
-            return index;
+        for (let i = 0; i < this.tabList_.length; i++) {
```

Comment:
I don't think you need this change.

Reply:
This is what's causing #276, the `forEach` is returning undefined

Reply:
ahh gotcha - lets just leave a comment so no one changes it back to a forEach or something